src/ralph/accounts/management/commands/ldap_sync.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+import sys
+import logging
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from ldap.controls import SimplePagedResultsControl
+
+
+logger = logging.getLogger(__name__)
+LDAP_RESULTS_PAGE_SIZE = 100
+
+try:
+    import ldap
+    from django_auth_ldap.backend import _LDAPUser
+    ldap_module_exists = True
+except ImportError:
+    logger.debug("Install python-ldap and django_auth_ldap packages")
```

Comment:
this is not necessary IMO - we have logger.error in handle when someone will run this command directly